src/main/java/seedu/address/ui/OrderCard.java

```diff
@@ -33,6 +36,8 @@ public class OrderCard extends UiPart<Region> {
     @FXML
     private Label orderId;
     @FXML
+    private Label orderCustomerName;
```

Comment:
Do stick with the naming convention, and change orderCustomerName to orderClientName

Reply:
Understood, thank you for catching this error. I will make the amendments immediately.